po/en.po

```diff
@@ -69,50 +69,66 @@ msgstr "Before the Cursor (Like Hanin)"
 msgid "after_cursor"
 msgstr "After the Cursor (Like MS IME)"
 
-#: src/McBopomofo.h:80
-msgid "Bopomofo Keyboard Layout"
+#: src/McBopomofo.h:76
+msgid "Directly Output Uppercased Letters"
+msgstr ""
+
+#: src/McBopomofo.h:77
+msgid "Put Lowercased Letters to Composing Buffer"
 msgstr ""
 
 #: src/McBopomofo.h:85
-msgid "Selection Keys"
+msgid "Bopomofo Keyboard Layout"
 msgstr ""
 
 #: src/McBopomofo.h:90
+msgid "Selection Keys"
+msgstr ""
+
+#: src/McBopomofo.h:95
 msgid "Show Candidate Phrase"
 msgstr "Show Candidates"
 
-#: src/McBopomofo.h:94
+#: src/McBopomofo.h:100
 msgid "Move cursor after selection"
 msgstr ""
 
-#: src/McBopomofo.h:102
-msgid "Map Dvorak to QWERTY"
+#: src/McBopomofo.h:106
+msgid "ESC key clears entire composing buffer"
+msgstr ""
+
+#: src/McBopomofo.h:110
+msgid "Shift + Letter Keys"
 msgstr ""
 
-#: src/KeyHandler.cpp:519
+#: src/KeyHandler.cpp:581
 msgid "Cursor is between syllables {0} and {1}"
 msgstr ""
 
-#: src/KeyHandler.cpp:611
+#: src/KeyHandler.cpp:673
 msgid "{0} syllables required"
 msgstr ""
 
-#: src/KeyHandler.cpp:614
+#: src/KeyHandler.cpp:676
 msgid "{0} syllables maximum"
 msgstr ""
 
-#: src/KeyHandler.cpp:617
+#: src/KeyHandler.cpp:679
 msgid "phrase already exists"
 msgstr ""
 
-#: src/KeyHandler.cpp:619
+#: src/KeyHandler.cpp:681
 msgid "press Enter to add the phrase"
 msgstr ""
 
-#: src/KeyHandler.cpp:623
+#: src/KeyHandler.cpp:685
 msgid "Marked: {0}, syllables: {1}, {2}"
 msgstr ""
 
 #: src/mcbopomofo.conf.in.in:3 src/mcbopomofo-addon.conf.in.in:3
 msgid "McBopomofo"
 msgstr ""
+
+#: src/mcbopomofo-addon.conf.in.in:4
+msgid "McBopomofo Wrapper For Fcitx"
```

Comment:
Do other addons use the term "wrapper"? Or should we just call this "McBopomofo for fcitx5"?

Also commented on the addon conf.